### PR TITLE
add doc for exportData

### PR DESCRIPTION
**Pull request by eheckel**

### CODE REVIEW

File: documentation/docs/api/ObjectHandle.md [hunk 1]
```diff
@@ -13,6 +13,19 @@ discarded.
 
 ## Methods
 
+### exportData(format: string, options?: ExportDataOptions): Promise\<string>
+Exports a file containing the object's data, returning a URL to the file.
```
Comment (joylashley) [inline documentation/docs/api/ObjectHandle.md:17]: Exports a file that contains the object's data, and returns a URL for the file.

File: documentation/docs/api/ExportDataOptions.md [hunk 2]
```diff
@@ -0,0 +1,31 @@
+---
+id: ExportDataOptions
+title: ExportDataOptions
+---
+
+ExportDataOptions provides options for customizing the properties of exported report data. If no option is specified, then the default value is used.
```
Comment (joylashley) [inline documentation/docs/api/ExportDataOptions.md:6]: ...the properties of the report data that you want to export.

File: documentation/docs/api/ExportDataOptions.md [hunk 3]
```diff
@@ -0,0 +1,31 @@
...
+
+default value: `<index of last row of data>`
+
+### `formattedData: boolean` 
+Specifies whether the data is formatted.
+
+default value: `true`
+
+### `detailedData: boolean` 
+Specifies whether the data is detailed.
```
Comment (brmorr) [inline documentation/docs/api/ExportDataOptions.md:29]: I think we need to be more descriptive about what "detailed" data is.  @joylashley, how is this option described by the VA export data UI doc?

Comment (brmorr) [reply documentation/docs/api/ExportDataOptions.md:29]: I think the description of detailedData would mention aggregated vs unaggregated data, and its effect on the available rows and columns.  Discussing it there seems sufficient.

Comment (joylashley) [reply documentation/docs/api/ExportDataOptions.md:29]: I looked at our definition of aggregated data sources in another VA doc. 

Would this work here: Specifies whether the data is aggregated (which has fewer columns, rows, or both depending on filters that have been applied) or detailed (which has all available rows and columns).

Comment (brmorr) [reply documentation/docs/api/ExportDataOptions.md:29]: that works for me

Comment (joylashley) [reply documentation/docs/api/ExportDataOptions.md:29]: Oops! I just noticed I didn't make the end of that sentence consistent with the beginning. Please change ending to "... or detailed (which has all available **columns and rows**).

Comment (eheckel) [reply documentation/docs/api/ExportDataOptions.md:29]: Sounds good. Is formattedData descriptive enough? 

In VA's doc, its description is "Controls whether formatted or unformatted data is exported. For example, if a data item has a dollar format applied, then exporting the formatted data includes the dollar sign. If this option is not selected, then the data for that item is exported as a raw number."

Comment (brmorr) [reply documentation/docs/api/ExportDataOptions.md:29]: I'm not opposed to making this one more descriptive as well.  I like the sound of what VA has.

Comment (joylashley) [reply documentation/docs/api/ExportDataOptions.md:29]: I like reusing the VA doc description for formattedData too.

Comment (eheckel) [reply documentation/docs/api/ExportDataOptions.md]: I modified the VA description slightly to swap "option is not selected" to "option is set to false" to work with our API.

Comment (joylashley) [reply documentation/docs/api/ExportDataOptions.md]: Sounds good, @eheckel.

**Pull request merged**